.flake8

```diff
@@ -1,5 +1,5 @@
 [flake8]
-ignore = E203, E266, E501, W503
+extend-ignore = E203, E266, E501
 # line length is intentionally set to 80 here because black uses Bugbear
 # See https://github.com/psf/black/blob/master/README.md#line-length for more details
```

Comment:
This link doesn't make sense anymore since the docs were refactored.

```suggestion
# See https://github.com/psf/black/blob/master/docs/the_black_code_style.md#line-length for more details```

Reply:
Also since [you can't comment on non modified code using the PR review UI 🙁](https://github.com/isaacs/github/issues/284), please also update the link at line 35 of `docs/compatible_configs.md` to `https://github.com/psf/black/blob/master/docs/the_black_code_style.md#how-black-wraps-lines`.

Reply:
Thanks for the review. I applied the suggested changes.